certs/certs.go

```diff
@@ -34,6 +35,60 @@ import (
 	"sigs.k8s.io/etcdadm/constants"
 )
 
+// GetDefaultCertList returns all of the certificates etcdadm requires to function.
+func GetDefaultCertList() []string {
+	return []string{
+		//constants.EtcdCACertAndKeyBaseName,
+		constants.EtcdServerCertAndKeyBaseName,
```

Comment:
How does replacing a certificate affect the client that uses the certificates?

1. etcd peer and server certificates: etcd will check and, if needed, [reload the certificate on every connection](https://github.com/etcd-io/etcd/pull/7829).
2. etcdctl client certificate: this is not a long-running process, and presumably etcdctl reads certificates when it starts. If it fails, the user can always rerun it.
3. **kube-apiserver: I'm not sure if it will reload its etcd client certificate. Does it need to be restarted? **

Reply:
1. Etcd should restart as far as I know.

2. etcdctl every time use the new certificate s , so I think renew not effect it.

3. If the kube-apisever client crt changed, I think kube-apisever maybe need to restart. I'm not tests this case, so i'm not sure. I will tests it.

Reply:
Thank you!

If kube-apiserver does need to be restarted, let's document that. Perhaps even write a message to the user.